README.Rmd

```diff
@@ -70,7 +69,7 @@ knitr::include_graphics("man/figures/auto-after.png")
 
 Instead of relying on **thematic** to automatically detect colors and fonts in the plot's container, you can also specify them directly in `thematic_on()`.
 
-```r
+```{r}
 thematic_on(bg = "#222222", fg = "white", accent = "#0CE3AC", font = "Oxanium")
```

Comment:
I enjoy seeing library statements to know that it was loaded. (Feel free to reject this `suggestion`)

```suggestion
library(thematic)
thematic_on(bg = "#222222", fg = "white", accent = "#0CE3AC", font = "Oxanium")
```